tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "lib": ["es2015"],
+    "target": "es2015",
+    "module": "commonjs",
```

Comment:
nit: I believe these these three lines can be omitted because they are the same as `tsconfig-google.json`?

Reply:
`lib` and `module` are the same -- `target` is actually not since we want to preserve generators for the Koa plugin.